client/components/authentication/Login.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react'
+import axios from 'axios'
+import UserInfo from './UserInfo'
+
+function Login () {
+  return(
+    <>
+      <h1>Login</h1>
+      <UserInfo type='login'></UserInfo>
+    </>
+  )
+}
+
+export default Login
```

Comment:
minor optimization note: if Login is the only thing we're exporting from this file, I think we can just add "export default" at the beginning of Line 5